odoo/models.py

```diff
@@ -1542,7 +1542,7 @@ def name_get(self):
 
     @api.model
     def name_create(self, name):
-        """ name_create(name) -> record
+        """ name_create(name) -> (int, str)
```

Comment:
```suggestion
        """ name_create(name) -> (int, str, bool)
```
if it's returning False 🤔

Reply:
It is 😅 but those aren't real type hints. It's just to indicate that we return a tuple with the record id and its display name. As a type hint it would be something like this

```suggestion
        """ name_create(name) -> tuple[int, str] | bool
```

In any case, I kept the scenario where `False` is returned when it's not possible to create the record, but it actually seems like a very error-prone idea. We should probably raise an exception there, instead.. 🤔